cmake/modules/FindALUMINUM.cmake

```diff
@@ -0,0 +1,52 @@
+# Defines the following variables:
+#   - ALUMINUM_FOUND
+#   - ALUMINUM_LIBRARIES
+#   - ALUMINUM_INCLUDE_DIRS
+#
+# Also creates an imported target ALUMINUM
+
+# Find the header
+find_path(ALUMINUM_INCLUDE_DIRS common.h
+  HINTS ${ALUMINUM_DIR} $ENV{ALUMINUM_DIR}
+  PATH_SUFFIXES include
+  NO_DEFAULT_PATH
+  DOC "Directory with ALUMINUM header.")
+find_path(ALUMINUM_INCLUDE_DIRS common.h)
+
+# Find the library
+find_library(ALUMINUM_LIBRARY liballreduce.so
```

Comment:
Don't need the ".so" at the end or the "lib" at the front. CMake knows to add those, so you can just do ```find_library(ALUMINUM_LIBRARY allreduce)```